### ISSUE TITLE
How to cooperate with YMF825 hardware?

### ISSUE THREAD

**Issue by denjhang** [danielrfry/notesalad-editor#8]
Is the hardware here the same as opl2sd1? If the same, where is the firmware?

**Comment by danielrfry** [comment]
Yes, it will work with the same hardware. I didn't initially publish the firmware, as I wasn't sure how much interest there would be in the YMF825 support, but I'll clean up and add the code I use as an example.
